src/lib/components/archiveProject.svelte

```diff
@@ -0,0 +1,295 @@
+                                                on:click={() => handleUnarchiveProject(project)}
+                                                >Unarchive project</ActionMenu.Item.Button>
+                                            <ActionMenu.Item.Button
+                                                leadingIcon={IconSwitchHorizontal}
+                                                on:click={() => handleMigrateProject(project)}
+                                                >Migrate project</ActionMenu.Item.Button>
+                                        </ActionMenu.Root>
+                                    </Popover>
+                                </div>
+                            </svelte:fragment>
+
+                            {#each platforms.slice(0, 2) as platform}
+                                {@const icon = getIconForPlatform(platform.icon)}
+                                <Badge
+                                    variant="secondary"
+                                    content={platform.name}
+                                    style="width: max-content;">
+                                    <Icon {icon} size="s" slot="start" />
+                                </Badge>
+                            {/each}
+
+                            {#if platforms.length > 3}
+                                <Badge
+                                    variant="secondary"
+                                    content={`+${platforms.length - 2}`}
+                                    style="width: max-content;" />
+                            {/if}
+
+                            <svelte:fragment slot="icons">
+                                {#if isCloud && $regionsStore?.regions}
+                                    {@const region = findRegion(project)}
+                                    <Typography.Text>{region.name}</Typography.Text>
+                                {/if}
+                            </svelte:fragment>
+                        </GridItem1>
+                    {/each}
+                </CardContainer>
+            </div>
+        </Accordion>
+    </div>
+{/if}
+
+<!-- Unarchive Confirmation Modal -->
+<Modal bind:show={showUnarchiveModal} title="Unarchive project">
+    <div class="modal-content">
+        <p>Are you sure you want to unarchive <strong>{projectToUnarchive?.name}</strong>?</p>
+        <p>This will move the project back to your active projects list.</p>
+    </div>
+
+    <svelte:fragment slot="actions">
+        <Button secondary on:click={cancelUnarchive}>Cancel</Button>
+        <Button on:click={confirmUnarchive}>Unarchive</Button>
+    </svelte:fragment>
+</Modal>
+
+<style>
+    .status-container {
+        display: flex;
+        align-items: center;
+        gap: 8px;
+    }
+
+    .modal-content {
+        padding: var(--space-4) 0;
+    }
+
+    .modal-content p {
+        margin-bottom: var(--space-3);
+    }
```

Comment:
why do we need custom dimensions here? Modal has relevant theme/styles by default afaik.